packages/calling-component-bindings/src/captionsSelector.ts

```diff
@@ -122,6 +122,16 @@ export const _captionsBannerSelector: _CaptionsBannerSelector = reselect.createS
     // Following Teams app logic, no matter how many 'Partial' captions come,
     // we only pick first one according to start time, and all the other partial captions will be filtered out
     // This will give customers a stable captions experience when others talking over the dominant speaker
+    // First turn all partial captions that are older than 10 seconds to final
```

Comment:
Is there a work item filed on the backend to fix these unfinalized captions?

Reply:
Looks like this is happening at the root from teams caption bot, teams has the same logic to fix the issue on UI end

Reply:
Could we move this fix to the calling SDK then? Otherwise, how will external developers know to implement a 5-second workaround to not hit the same bug

Reply:
Captions data is not stored on sdk, we would have to have temporary store captions in the sdk if we want to have the fix on SDK.
This is probably one of the reasons why Teams does it on the ui as well.